### ISSUE TITLE
About adding label with index 0 for some examples

### ISSUE THREAD

**Issue by dhruvdcoder** [Jiankai-Sun/plate-ral#3]
In line 437 below, you seem to append label with index 0 for videos that have number of steps less than max (3 or 4). As far as I can tell, the index 0 is not reserved for a special action. Moreover, you don't have 0 in the beginning for the examples that have number of steps greater than max trajectory (3 or 4). May I know the reason for adding the label with index 0 for some but not other examples?
Thanks!

https://github.com/Jiankai-Sun/plate-pytorch/blob/c3888270611740c4012d0912332405f64d81586d/data_utils/data_loader_frames.py#L424-L437

**Comment by Jiankai-Sun** [comment]
Good point! This is an imperfect approximation to the corner case of "the number of steps is less than the maximum trajectory" examples. The reason for adding the label with index 0 is that we can't get the corresponding label from the label matrix, whereas for the other examples, we can get the ground-truth label from the label_matrix provided by CrossTask Dataset.
Maybe it would be a better choice to ignore the "number of steps is less than the maximum trajectory" examples?
